Makefile

```diff
@@ -1,26 +1,26 @@
 .PHONY: build release install uninstall clean test doc reindent
 
 build:
-	jbuilder build @install --dev
+	dune build @install
 
 release:
-	jbuilder build @install
+	dune build @install
```

Comment:
You can use --profile=release, I think dune defaults to using --dev, which have warnings-as-errors, and also -opaque which inhibits cross-file optimizations in ocamlopt. Using `--profile=release` (or `-p <pkg>`) restores the optimized behaviour.